cloud: add use TiDB Cloud Starter with AI Tools (#22228)
pingcap/docs: tidb-cloud/tidb-cloud-use-with-ai-tools.md

    2. If you have already set a password:
        1. You can enter your password in the **Enter the password for easy setup** input box.
        2. If you forget your password, clicking **Reset password** in the **Prerequisites** section to get a new password. Be aware that resetting the password will drop root's connection to the current cluster.

5. Switch to the tab you want: **Cursor**, **Claude Code**, **VS Code**, or **Windsurf**.
6. Follow the tool-specific setup steps in the selected tab.

## Tool-specific setup

### Cursor

1. Click **Add to Cursor** to open Cursor, and then click **Install** to finish the setup.
2. Or add the configuration to `.cursor/mcp.json`, as follows:

```json
{
  "mcpServers": {
    "TiDB": {
      "command": "uvx --from pytidb[mcp] tidb-mcp-server",
      "env": {
        "TIDB_HOST": "<YOUR_TIDB_HOST>",
        "TIDB_PORT": "<YOUR_TIDB_PORT>",
        "TIDB_USERNAME": "<YOUR_TIDB_USERNAME>",
        "TIDB_PASSWORD": "<YOUR_TIDB_PASSWORD>",
        "TIDB_DATABASE": "<YOUR_TIDB_DATABASE>"
      }
    }
  }
}
```

### Claude Code

1. Click the first copy button to copy the command.

> 2. Open the terminal and add the MCP server by running the copied command.

Author: Sure, added.

> 3. Alternatively, follow Claude Code's documentation to add the configuration to `.mcp.json` for project scope, as follows:

Reviewer: ```suggestion
3. Alternatively, follow [Claude Code's documentation](https://code.claude.com/docs/en/mcp#project-scope) to add the configuration to `.mcp.json` for project scope, as follows:
```

---
title: Use {{{ .starter }}} with AI Tools
summary: Learn how to use your {{{ .starter }}} cluster with AI tools.
---

> # Connect to Your {{{ .starter }}}

Reviewer: +1

This document describes how to use your {{{ .starter }}} cluster with AI tools.

## Steps

After your {{{ .starter }}} cluster is created on TiDB Cloud, you can use it with AI tools via the following steps:

1. On the [**Clusters**](https://tidbcloud.com/project/clusters) page, click a cluster name to go to its overview page, and then click **Use with AI Tools** at the top of the page.
2. In the **Access `your_cluster_name` with AI tools** dialog, select the **Branch** and **Database** that you want to connect to in the AI tool.
3. Ensure that you meet the **Prerequisites**. If not, follow the instructions on the page to install the required dependencies.
4. For the password:

    1. If you have not set a password yet, click **Generate Password** to generate a random password. The generated password will not be shown again, so save your password in a secure location.
    2. If you have already set a password, you can:

        1. Click **Reset password** in the **Prerequisites** section. This resets the password of the `root` user and might break existing connections to this cluster that use `root`.

>         2. You can enter your password in the **Enter the password for easy setup** input box.

Reviewer: ```suggestion
    2. If you have already set a password:
        1. You can enter your password in the **Enter the password for easy setup** input box.
        2. If you forget your password, clicking **Reset password** in the **Prerequisites** section to get a new password. Be aware that resetting the password will drop root's connection to the current cluster.
```

```json
{
  "mcpServers": {
    "TiDB": {
      "command": "uvx --from pytidb[mcp] tidb-mcp-server",
      "env": {
        "TIDB_HOST": "<YOUR_TIDB_HOST>",
        "TIDB_PORT": "<YOUR_TIDB_PORT>",
        "TIDB_USERNAME": "<YOUR_TIDB_USERNAME>",
        "TIDB_PASSWORD": "<YOUR_TIDB_PASSWORD>",
        "TIDB_DATABASE": "<YOUR_TIDB_DATABASE>"
      }
    }
  }
}
```

### Claude Code

1. Click the first copy button to copy the command.
2. Open the terminal and add the MCP server by running the copied command.
3. Alternatively, follow Claude Code's documentation to add the configuration to `.mcp.json` for project scope, like below:

```json
{
  "mcpServers": {
    "TiDB": {
      "type": "stdio",
      "command": "uvx",
      "args": ["--from", "pytidb[mcp]", "tidb-mcp-server"],
      "env": {
        "TIDB_HOST": "<YOUR_TIDB_HOST>",
        "TIDB_PORT": "<YOUR_TIDB_PORT>",
        "TIDB_USERNAME": "<YOUR_TIDB_USERNAME>",
        "TIDB_PASSWORD": "<YOUR_TIDB_PASSWORD>",
        "TIDB_DATABASE": "<YOUR_TIDB_DATABASE>"
      }
    }
  }
}
```

### VS Code

1. Click **Add to VS Code** to open VS Code, and then click **Install** to finish the setup.

> 2. Or add the configuration to `.vscode/mcp.json`, like below:

Author: ```suggestion
2. Or add the configuration to `.vscode/mcp.json`, as follows:
```

```json
{
  "mcpServers": {
    "TiDB": {
      "type": "stdio",
      "command": "uvx",
      "args": ["--from", "pytidb[mcp]", "tidb-mcp-server"],
      "env": {
        "TIDB_HOST": "<YOUR_TIDB_HOST>",
        "TIDB_PORT": "<YOUR_TIDB_PORT>",
        "TIDB_USERNAME": "<YOUR_TIDB_USERNAME>",
        "TIDB_PASSWORD": "<YOUR_TIDB_PASSWORD>",
        "TIDB_DATABASE": "<YOUR_TIDB_DATABASE>"
      }
    }
  }
}
```

### Windsurf

> Follow Windsurf's documentation to add the configuration to `mcp_config.json`, like below:

Author: ```suggestion
Follow Windsurf's documentation to add the configuration to `mcp_config.json`, as follows:
```

---
title: Use {{{ .starter }}} with AI tools

> summary: Learn how to use your {{{ .starter }}} cluster with AI tools.

Reviewer 2: ```suggestion
title: Use {{{ .starter }}} with AI Tools
summary: Learn how to connect your {{{ .starter }}} cluster to AI-powered development tools that support the Model Context Protocol (MCP), such as Cursor, Claude Code, VS Code, and Windsurf.
```

---

> # Use {{{ .starter }}} with AI tools

Reviewer 2: ```suggestion
# Use {{{ .starter }}} with AI Tools
```
Use title case

> This document describes how to use your {{{ .starter }}} cluster with AI tools.

Reviewer 2: ```suggestion
This document describes how to connect your {{{ .starter }}} cluster to AI-powered development tools that support the Model Context Protocol (MCP), such as Cursor, Claude Code, Visual Studio Code (VS Code), and Windsurf.

By configuring your TiDB cluster as an MCP server, you can enable AI assistants in your development tools to query your database schema, understand your data model, and generate context-aware code suggestions.
```

Clarify the document's scope and value by being more specific about what "AI tools" means and the practical benefits to users.

> 

Reviewer 2: ```suggestion

## Before you begin

To complete this guide, you need the following:

- A {{{ .starter }}} cluster. If you don't have any, you can [create a {{{ .starter }}} cluster](/develop/dev-guide-build-cluster-in-cloud.md).
- [Python 3.11 or higher](https://www.python.org/downloads/) installed.
- [uv](https://docs.astral.sh/uv/getting-started/installation/) installed.
- An AI development tool that supports MCP, such as:

    - [Cursor](https://cursor.com)
    - [Claude Code](https://claude.com/product/claude-code)
    - [Visual Studio Code](https://code.visualstudio.com)
    - [Windsurf](https://windsurf.com)

```
Add a clear, upfront checklist of prerequisites required to follow the guide successfully.

> ## Steps

Reviewer 2: ```suggestion
## Connect to AI tools
```

> After your {{{ .starter }}} cluster is created on TiDB Cloud, you can use it with AI tools via the following steps:

Reviewer 2: ```suggestion
After you create a {{{ .starter }}} cluster in TiDB Cloud, perform the following steps to connect it to your AI tool.
```

1. On the [**Clusters**](https://tidbcloud.com/project/clusters) page, click a cluster name to go to its overview page, and then click **Use with AI Tools** at the top of the page.
2. In the **Access `your_cluster_name` with AI tools** dialog, select the **Branch** and **Database** that you want to connect to in the AI tool.
3. Ensure that you meet the **Prerequisites**. If not, follow the instructions on the page to install the required dependencies.
4. For the password:

    1. If you have not set a password yet, click **Generate Password** to generate a random password. The generated password will not be shown again, so save your password in a secure location.
    2. If you have already set a password:
        1. You can enter your password in the **Enter the password for easy setup** input box.

>         2. If you forget your password, clicking **Reset password** in the **Prerequisites** section to get a new password. Be aware that resetting the password will drop root's connection to the current cluster.

Reviewer 2: ```suggestion
4. Configure the password:

    - If you have not set a password yet, click **Generate Password** to generate a random password.

        The generated password will not show again, so save your password in a secure location.

    - If you have already set a password, enter your password in the **Enter the password for easy setup** field.
    - If you forget the password, click **Reset password** in the **Prerequisites** section to generate a new one.

        Note that resetting your password disconnects all existing root user sessions.
```

5. Switch to the tab you want: **Cursor**, **Claude Code**, **VS Code**, or **Windsurf**.

> 6. Follow the tool-specific setup steps in the selected tab.

Reviewer 2: ```suggestion
5. Select the tab for your AI tool: **Cursor**, **Claude Code**, **VS Code**, or **Windsurf**.
6. Complete the setup steps for the selected tool.

    For more information, see [Tool-specific setup](#tool-specific-setup).
```

## Tool-specific setup

### Cursor

1. Click **Add to Cursor** to open Cursor, and then click **Install** to finish the setup.

> 2. Or add the configuration to `.cursor/mcp.json`, as follows:

Reviewer 2: ```suggestion
To configure Cursor as an MCP client for TiDB, you can use one of the following methods:

- **Method 1**: in the TiDB Cloud console, click **Add to Cursor** to launch Cursor, and then click **Install**.
- **Method 2**: manually add the following configuration to your `.cursor/mcp.json` file:
```

    ```json
    {
    "mcpServers": {
        "TiDB": {
        "command": "uvx --from pytidb[mcp] tidb-mcp-server",
        "env": {
            "TIDB_HOST": "<YOUR_TIDB_HOST>",
            "TIDB_PORT": "<YOUR_TIDB_PORT>",
            "TIDB_USERNAME": "<YOUR_TIDB_USERNAME>",
            "TIDB_PASSWORD": "<YOUR_TIDB_PASSWORD>",
            "TIDB_DATABASE": "<YOUR_TIDB_DATABASE>"
        }
        }
    }
    }
    ```

### Claude Code

1. Click the first copy button to copy the command.

> 2. Open the terminal and add the MCP server by running the copied command, as follows:

Reviewer 2: ```suggestion
To configure Claude Code as an MCP client for TiDB, you can use one of the following methods:

- **Method 1**: copy the setup command from the TiDB Cloud console and run it in your terminal:
```

    ```bash
    claude mcp add --transport stdio TiDB \
      --env TIDB_HOST='<YOUR_TIDB_HOST>' \
      --env TIDB_PORT=<YOUR_TIDB_PORT> \
      --env TIDB_USERNAME='<YOUR_TIDB_USERNAME>' \
      --env TIDB_PASSWORD='<YOUR_TIDB_PASSWORD>' \
      --env TIDB_DATABASE='<YOUR_TIDB_DATABASE>' \
      -- uvx --from 'pytidb[mcp]' 'tidb-mcp-server'
    ```

> 3. Alternatively, follow [Claude Code's documentation](https://code.claude.com/docs/en/mcp#project-scope) to add the configuration to `.mcp.json` for project scope, as follows:

Reviewer 2: ```suggestion
- **Method 2**: add the following configuration to your project-level `.mcp.json` file. For more information, see the [Claude Code documentation](https://code.claude.com/docs/en/mcp#project-scope).
```

        Note that resetting your password disconnects all existing root user sessions.

5. Select the tab for your AI tool: **Cursor**, **Claude Code**, **VS Code**, or **Windsurf**.
6. Complete the setup steps for the selected tool.

    For more information, see [Tool-specific setup](#tool-specific-setup).

## Tool-specific setup

### Cursor

To configure Cursor as an MCP client for TiDB, you can use one of the following methods:

- **Method 1**: in the TiDB Cloud console, click **Add to Cursor** to launch Cursor, and then click **Install**.
- **Method 2**: manually add the following configuration to your `.cursor/mcp.json` file:

    ```json
    {
      "mcpServers": {
        "TiDB": {
          "command": "uvx --from pytidb[mcp] tidb-mcp-server",
          "env": {
            "TIDB_HOST": "<YOUR_TIDB_HOST>",
            "TIDB_PORT": "<YOUR_TIDB_PORT>",
            "TIDB_USERNAME": "<YOUR_TIDB_USERNAME>",
            "TIDB_PASSWORD": "<YOUR_TIDB_PASSWORD>",
            "TIDB_DATABASE": "<YOUR_TIDB_DATABASE>"
          }
        }
      }
    }
    ```

### Claude Code

To configure Claude Code as an MCP client for TiDB, you can use one of the following methods:

> - **Method 1**: copy the setup command from the TiDB Cloud console and run it in your terminal:

Reviewer 3: ```suggestion
- **Method 1**: copy the setup command from the [TiDB Cloud console](https://tidbcloud.com/) and run it in your terminal:
```
